airbyte-cdk/bulk/build.gradle

```diff
@@ -61,6 +61,12 @@ allprojects {
     }
 }
 
+tasks.register('bulkCdkIntegrationTest').configure {
+    // findByName returns the task, or null if no such task exists.
+    // we need this because not all submodules have an integrationTest task.
```

Comment:
FYI, alternatively, `tasks.matching` is also useful for this purpose, perhaps even preferred